loc/items/sword.js

```diff
@@ -156,7 +156,7 @@ export default [
     description: {
       en: 'It can cut demons, but not angels',
       ru: 'Oно может резать демонов, но не ангелов',
-      es: 'Puede cortar demonios, pero no ángeles.',
+      es: 'San Dek lo usó en las guerras en los cielos.',
```

Comment:
Please dont diverge from the other texts to much, "Dek" is not part of the lore :) If you have questions you best ask dhwty in discord about these kind of things.
Other than that I think its great and good to go.

Reply:
fixed the problem